src/net.rs

```diff
@@ -62,6 +62,7 @@ impl Async<TcpListener> {
         socket.set_nonblocking(true)?;
         socket.set_reuse_address(true)?;
         socket.set_reuse_port(true)?;
+        socket.set_nodelay(true)?;
```

Comment:
I think Nagle is very suitable for situations where network bandwidth is limited, but it will lead to increased latency. I think we can use a feature: `#[cfg(feature = "net_nagle")]` for conditional compilation.